include/exadg/incompressible_navier_stokes/spatial_discretization/operator_coupled.cpp

```diff
@@ -171,11 +171,13 @@ unsigned int
 OperatorCoupled<dim, Number>::solve_linear_stokes_problem(BlockVectorType &       dst,
                                                           BlockVectorType const & src,
                                                           bool const &   update_preconditioner,
-                                                          double const & time,
                                                           double const & scaling_factor_mass)
 {
-  // Update linear operator
-  linear_operator.update(time, scaling_factor_mass);
```

Comment:
This line did actually not call `momentum_operator->set_scaling_factor_mass_operator()` in the previous implementation, such that the following line `linear_solver->update_preconditioner()` used a potentially wrong `scaling_factor_mass`. This can have a significant influence on iteration counts.